### PR TITLE
msz

### PR DESCRIPTION
**Pull request by rahuTi**
xtended

### CODE REVIEW

File: c1.txt [hunk 1]
```diff
@@ -2,4 +2,7 @@ this is my code...
 ...
 ...
 how about it.
-added aeede
\ No newline at end of file
+added aeede
+------------
+added portions
+------------
```
Comment (rahuTi) [inline c1.txt:8]: checking

Comment (rahuTi) [inline c1.txt:8]: 6-8

Comment (Rahu09) [inline c1.txt:8]: 6--8

Comment (Rahu09) [inline c1.txt:8]: 6---8

Comment (rahuTi) [inline c1.txt:8]: 5-8

Comment (rahuTi) [inline c1.txt:8]: 5-8

Comment (rahuTi) [inline c1.txt:8]: 5--8

Comment (rahuTi) [inline c1.txt:8]: 5---8

Comment (rahuTi) [inline c1.txt:8]: 2-8

Comment (rahuTi) [inline c1.txt:8]: 2+8

Comment (rahuTi) [inline c1.txt:8]: 2-8

Comment (rahuTi) [inline c1.txt:8]: 2------8

Comment (rahuTi) [inline c1.txt:8]: 3333

Comment (rahuTi) [inline c1.txt:8]: se

Comment (rahuTi) [inline c1.txt:8]: 2-8

Comment (rahuTi) [inline c1.txt:8]: 222

Comment (rahuTi) [inline c1.txt:8]: asa

Comment (rahuTi) [inline c1.txt:8]: 2-888

Comment (rahuTi) [inline c1.txt:8]: new one

Comment (rahuTi) [inline c1.txt:8]: nn

Comment (rahuTi) [inline c1.txt:8]: cccc

Comment (rahuTi) [inline c1.txt:8]: safsf

**Pull request closed without merging**